pandas/tests/plotting/test_quarterly_finder.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import pandas as pd
+import pytest
+from pandas.plotting._matplotlib import converter
+
+
+@pytest.mark.parametrize("year_span", [11.25, 30, 80, 150, 300, 584.5])
+# valid ranges are from 11.25 to 584.5 years, limited at the bottom by
+# if statements in the _quarterly_finder() function and at the top end
+# by the Timestamp format
+def test_quarterly_finder(year_span):
```

Comment:
1. Could you put this test in `pandas/tests/plotting/test_converter.py`?
2. Can this test be simplified at all?

Reply:
I moved the test over to that file and also simplified the initial test data generation a bit. I'm not sure how to simplify the test further without losing coverage or sacrificing clarity. Open to guidance, however.